src/config.js

```diff
@@ -3,31 +3,39 @@ import path from 'path'
 import fs from 'fs'
 import os from 'os'
 import isDev from 'electron-is-dev'
-import {app} from 'electron'
+import {app, dialog} from 'electron'
 import FileHistory from './utils/file-history'
 
-export const logoIpfsIce = (() => {
-  const p = path.resolve(path.join(__dirname, 'img'))
-
-  if (os.platform() === 'darwin') {
-    return path.join(p, 'icons/ice.png')
-  }
+// Set up what to do on Uncaught Exceptions
+process.on('uncaughtException', (error) => {
+  const msg = error.message || error
+  logger.error(`Uncaught Exception: ${msg}`, error)
+  dialog.showErrorBox('Uncaught Exception:', msg)
```

Comment:
This error box should guide the user to the issue tracker (this repo) and recommend to open a issue with the error

Reply:
Will do this tomorrow or sunday.